source/includes/_visitor_events.md

```diff
@@ -0,0 +1,174 @@
+### **Visitor Events**
+<a name="visitor_events"></a>
+
+### Resource Overview
+
+
+|Method|URI Format|
+|---|---|
+|GET|/client_reports/visitor_events?[query_params]|
+
+This API can be used to retrieve capture events for a list of visitor_ids for a gmaid.
+
+The data can be filtered with date range, and optionally grouped by visitor_id.
+
+When available, campaign data will be provided for each event, as well as channel when campaign type is xmedia.
+
+### Parameters
+
+When using the GET method, the results can be filtered using these parameters:
+
+|Param|Required|Function|
+|---|---|---|
+|gmaid|Yes|Restrict results to the specified Global Master Advertiser ID|
+|start_date|No|Restricts the results to those occurring on or after this date|
+|end_date|No|Restricts the results to those occurring on or before this date|
+|group_by|No|Only valid value at this time is `visitor_id`|
+
+### Response Data Details
+
+> Retrieve data for a list of event_ids
+
+```
+curl -H "Authorization: Bearer OAUTH_ACCESS_TOKEN" \
+https://api.reachlocalservices.com/client_reports/visitor_events?gmaid=TEST_1&visitor_ids[]='111-222-333'&visitor_ids[]='444-555-666'
+```
+
+> Retrieve data restricted by date_range
+
+```
+curl -H "Authorization: Bearer OAUTH_ACCESS_TOKEN" \
+https://api.reachlocalservices.com/client_reports/visitor_events?gmaid=TEST_1&visitor_ids[]='111-222-333'&visitor_ids[]='444-555-666'&start_date=2021-01-01&end_date=2021-01-31
+```
+
+> Retrieve data grouped by visitor_id
+
+```
+curl -H "Authorization: Bearer OAUTH_ACCESS_TOKEN" \
+https://api.reachlocalservices.com/client_reports/visitor_events?gmaid=TEST_1&visitor_ids[]='111-222-333'&visitor_ids[]='444-555-666'&group_by=visitor_id
+```
+> all filters combined (results below)
+
+```
+curl -H "Authorization: Bearer OAUTH_ACCESS_TOKEN" \
+https://api.reachlocalservices.com/client_reports/visitor_events?gmaid=TEST_1&visitor_ids[]='0b1f0b35-718a-4143-8ab6-5e15d380d6ad'&visitor_ids[]='499eea8d-dab5-46ec-8387-0b48a56c4dcb'&start_date=2021-01-01&end_date=2021-01-31&group_by=visitor_id
+```
+
+
+> Example Response
```

Comment:
The response structure changes if grouping by visitor_id is present.  This should be called out.  The example response shows only when grouping is active.